### PR TITLE
Various improvements to the docs and tests

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

File: tests/simple.rs [hunk 1]
```diff
@@ -1 +1,32 @@
+use chrono::{Duration, Local,Weekday};
+use parse_datetime::{parse_datetime_at_date, parse_relative_time::parse_relative_time, parse_timestamp::parse_timestamp, parse_weekday::parse_weekday};
```
Comment (sylvestre) [inline tests/simple.rs:2]: I am not a big fan of the parse_relative_time::parse_relative_time duplication
any idea how to improve this ?

**Comment by sylvestre** [comment]
@philolo1 feel free to review :)

**Comment by tertsdiepraam** [comment]
I don't think we should expose these functions. See my comment here: https://github.com/uutils/parse_datetime/pull/33#issuecomment-1732279797

**Comment by sylvestre** [comment]
makes sense, thanks :)

**Pull request closed without merging**